### PR TITLE
feat(wix-ui-tpa): Add Ratio prop to <Card />

### PR DESCRIPTION
**Pull request by bordi**
Add `Ratio` prop to the `<NewCard />`. 
Docs: https://docs.google.com/document/d/1O0HU4ZerEqF_Kjm7nhrsYdtg1FNteGg0O57vu2YvYrQ/edit?ts=6085ae53

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_7077/

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_7077

File: packages/wix-style-react/.wuf/components.json [hunk 1]
```diff
@@ -255,7 +255,10 @@
     "path": "src/InfoIcon"
   },
   "Input": {
-    "path": "src/Input"
+    "path": "src/Input",
+    "missingFiles": [
+      "docs/index.story.js"
+    ]
```
Comment (jonathanadler) [inline packages/wix-style-react/.wuf/components.json:261]: 🤔  I guess this is ok, but I wonder why this was changed

File: packages/wix-ui-tpa/src/components/NewCard/NewCard.st.css [hunk 2]
```diff
@@ -8,10 +23,12 @@
     flex-direction: column;
 }
 
+
 .container {
     flex: 1;
     max-height: 100%;
     box-sizing: border-box;
+    flex-basis: applyUnit(fallback(value(Ratio), value(DefalutRatio)), '%');
```
Comment (jonathanadler) [inline packages/wix-ui-tpa/src/components/NewCard/NewCard.st.css:31]: I'm actually not sure we want to apply percent.

I thought that maybe even passing one card container with `1` and another with `2` for instance would also work.
Theoretically 80 and 20 would work also without percentage, or even 8 and 2.

File: packages/wix-ui-tpa/src/components/Card/docs/index.story.tsx [hunk 3]
```diff
@@ -103,6 +103,22 @@ export default {
...
+            exampleStyles,
+          ),
+          example(
+            {
+              title: 'Ratio Example',
+              description: 'Card containers ratio can be changed 60/40',
+              source: examples.withRatioExample2,
+            },
+            exampleStyles,
+          ),
```
Comment (jonathanadler) [inline packages/wix-ui-tpa/src/components/Card/docs/index.story.tsx:121]: These are great,
But I guess we'll also need to add the `settingsPanel`
Like this for instance: https://github.com/wix/wix-style-react/blob/5e7de51ad016ffd3419d077e570195ba91895e49/packages/wix-ui-tpa/src/components/Badge/docs/index.story.tsx#L116.

Though I can also add it myself, so just if you think you can get to it as well.

**Review by jonathanadler — COMMENTED** [review]
Looks pretty awesome!

I left some small comments, would love to hear what you think.

Thanks 🚀

File: packages/wix-style-react/.wuf/components.json (hunk 1 above)
Comment (bordi) [reply packages/wix-style-react/.wuf/components.json:261]: It was after the `npm run reinstall`. 
Can remove this change if needed

File: packages/wix-ui-tpa/src/components/NewCard/NewCard.st.css [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+
+:vars {
+    /*Constance*/
+    DefalutRatio: 50;
+}
+
+:vars {
+    /*Overrides*/
+    Ratio: --overridable;
+}
```
Comment (jonathanadler) [inline packages/wix-ui-tpa/src/components/NewCard/NewCard.st.css:14]: Just a small small fix
```suggestion
/* Overrides */
:vars {
    /*  The ratio of a single card container */
    Ratio: --overridable;
}
```

**Review by jonathanadler — APPROVED** [review]
Approving, just left a teent tiny fix.

Nice job 💪

**Pull request merged**